h/presenters/annotation_searchindex.py

```diff
@@ -47,7 +47,7 @@ def asdict(self):
     def _add_hidden(self, result):
         # Mark an annotation as hidden if it and all of it's children have been
         # moderated and hidden.
-        parents_and_replies = [self.annotation.id] + self.annotation.thread_ids  # noqa: RUF005
+        parents_and_replies = [self.annotation.id, *self.annotation.thread_ids]
 
```

Comment:
TODO This method below is not reading from the new status column yet.

This has the effect that for example single annotation threads are not hidden from the response (but their content is hidden).